setup.py

```diff
@@ -12,9 +11,12 @@
         req for req in dev_requirements_file if not req.startswith("-")
     ]
 
+# get the version into a global variable named "version"
+exec(open("snowfakery/version.py").read())
```

Comment:
This doesn't look like it will work on Windows (slash separator, not being careful to close the opened file).

Could we copy CumulusCI's approach and just put the version in a .txt file, partly for consistency and partly to avoid needing to use exec?

Reply:
I did change it but just for interest's sake:

Slashes do work on Windows. And Windows will also close files automatically. The issue is actually that Windows hates to have an open file that is not in a directory whereas Unix doesn't care. So in Windows you must be careful to close files before deleting directories and therefore manual closing encourages you to think through the order of resource cleanup to make sure that the file handle goes away before the directory.